poetry.lock

```diff
@@ -924,7 +924,7 @@ bcrypt = ["bcrypt"]
 
 [[package]]
 name = "django-ansible-base"
-version = "2025.5.8"
+version = "25.7.21.0.dev768+g58a1b55"
```

Comment:
I don't know why this changed, and wonder if it was wrong before. It could have been due to packaging bugs on the DAB side, I don't know.